formatconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "exclude": [
+    "iron-fit-behavior.js",
```

Comment:
I think we need a way to tell the formatter to ignore certain parts of a file. This file should definitely be part of formatting but the formatter breaks a table in a comment here while trying to obey the line length limit.

Reply:
You can already do it by adding

Reply:
I shot a PR over to change the default

Reply:
Did we merge this default? It still seems to break tables

Reply:
I think we ended up saying that adding `formatconfig.json` was preferable.